### PR TITLE
feat(CategoryTheory/Monoidal): add lemmas for whiskering operators

### PR DESCRIPTION
**Pull request by yuma-mizuno**
Extracted from #6307 as suggested in https://github.com/leanprover-community/mathlib4/pull/6307#discussion_r1371849810 .

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by joelriou** [comment]
Thanks!

bors merge

File: Mathlib/RepresentationTheory/Action.lean [hunk 1]
```diff
@@ -509,6 +509,16 @@ theorem tensorHom {W X Y Z : Action V G} (f : W ⟶ X) (g : Y ⟶ Z) : (f ⊗ g)
 set_option linter.uppercaseLean3 false in
 #align Action.tensor_hom Action.tensorHom
 
+@[simp]
+theorem whiskerLeft_v (X : Action V G) {Y Z : Action V G} (f : Y ⟶ Z) :
```
Comment (eric-wieser) [inline Mathlib/RepresentationTheory/Action.lean:513]: I'm not sure the `v` is appropriate here, I'd expect `hom`

Comment (yuma-mizuno) [reply Mathlib/RepresentationTheory/Action.lean]: Thanks! I also renamed `tensorHom` to `tensor_hom` (probably a remnant from porting).

**Comment by eric-wieser** [comment]
bors merge

(I think bors is broken, but this will flag the PR as ready for when we sort it out)

Thanks!

**Pull request merged**